Describe automatic volume resizing in data studios (#335)
seqeralabs/docs: platform_versioned_docs/version-24.2/data_studios/index.mdx

The cleanup process is a best effort and not guaranteed. Seqera attempts to remove the checkpoint, but it can fail if, for example, the compute environment credentials used do not have sufficient permissions to delete objects from storage buckets.
:::

## Session volume automatic resizing

> By default, a data studio session allocates an initial 2 GB of storage. Available disk space is continually monitored and if the available space declines below a 1 GB threshold, Data Studios dynamically resizes the file system to include an additional 2 GB of available disk space.

Reviewer: ```suggestion
By default, a session allocates an initial 2 GB of storage. Available disk space is continually monitored and if the available space drops below a 1 GB threshold, the the file system is dynamically-resized to include an additional 2 GB of available disk space.
```

Reviewer 2: @justinegeffen You have two `the` next to each other?

This approach ensures that a session doesn't initially include unnecessary free disk space, while providing the flexibility to accommodate installation of large software packages required for data analysis.

> The maximum storage allocation for a session is limited by the compute environment disk boot size. By default, this is 30 GB. This limit is shared by all data studio sessions running on the same compute environment.

Reviewer: ```suggestion
The maximum storage allocation for a session is limited by the compute environment disk boot size. By default, this is 30 GB. This limit is shared by all sessions running on the same compute environment.
```

Reviewer 2: @justinegeffen: Update to "running **in** the same compute environment"

> If the maximize allocation size is reached, it is possible to reclaim storage space. To do so, stop the session, which triggers a snapshot from the active volume. Data Studios uploads the snapshot to cloud storage with Fusion. Starting from the newly saved snapshot loads all previous data from the snapshot. The newly started instance again has 2 GB of available space.

Reviewer: ```suggestion
If the maximum allocation size is reached, it is possible to reclaim storage space using a snapshot. 

Stop the active session to trigger a snapshot from the active volume. Data Studios uploads the snapshot to cloud storage with Fusion. When you start from the newly saved snapshot, all previous data is loaded. The newly-started session will have 2 GB of available space.
```

The cleanup process is a best effort and not guaranteed. Seqera attempts to remove the checkpoint, but it can fail if, for example, the compute environment credentials used do not have sufficient permissions to delete objects from storage buckets.
:::

## Session volume automatic resizing

By default, a session allocates an initial 2 GB of storage. Available disk space is continually monitored and if the available space drops below a 1 GB threshold, the file system is dynamically-resized to include an additional 2 GB of available disk space.

This approach ensures that a session doesn't initially include unnecessary free disk space, while providing the flexibility to accommodate installation of large software packages required for data analysis.

The maximum storage allocation for a session is limited by the compute environment disk boot size. By default, this is 30 GB. This limit is shared by all data studio sessions running on the same compute environment.

If the maximum allocation size is reached, it is possible to reclaim storage space using a snapshot. 

> Stop the active session to trigger a snapshot from the active volume. Data Studios uploads the snapshot to cloud storage with Fusion. When you start from the newly saved snapshot, all previous data is loaded. The newly-started session will have 2 GB of available space.

Reviewer: ```suggestion
Stop the active session to trigger a snapshot from the active volume. Data Studios uploads the snapshot to cloud storage with Fusion. When you start from the newly saved snapshot, all previous data is loaded and the newly-started session will have 2 GB of available space.
```
